resources/todomvc/big-dom-generator/src/components/tree-area.jsx

```diff
@@ -8,10 +9,14 @@ const TreeItem = (props) => {
     const { treeNode, currentDepth } = props;
 
     const isExpandableItem = treeNode.type === "expandableItem";
-    const treeViewItemIsOpen = isExpandableItem && currentDepth < MAX_VISIBLE_TREE_VIEW_ITEM_DEPTH ? "is-open" : "";
+    /**
+     * Every expandable TreeItem is open by default unless it is at the MAX_VISIBLE_TREE_VIEW_ITEM_DEPTH threshold.
+     * If it is marked as display none, set it as open so the `visibility: hidden` spectrum rule is removed.
```

Comment:
IMO this makes things a bit difficult to follow, what do you think about adding `visibility: unset` to the `.display-none` class?
BTW I think we also want to reset `height` to `auto` are we? Not sure this is necessary.

Reply:
This was just a cheap way of removing the `visibility: hidden` class from `.display-none` elements, doing `visibility: unset` inside the `.display-none` rule wouldn't work because the specificity of the original rule is higher. I think it should be fine to let `.display-none` elements have both `visibility: hidden` and `dispaly: none` properties, since `display: none` will take the elements out of the layout. Alternatively, I could use `!important` to unset them, but that seems even more artificial. WDYT?

Reply:
I think I'd use !important here, in this context, so that the 2 rules are in the same place. With some comment explaining why this is present.

Reply:
Done.